CMakeLists.txt

```diff
@@ -1087,7 +1087,11 @@ if(SDL_LIBC)
           if(PC_ICONV_FOUND)
             sdl_link_dependency(iconv LIBS Iconv::Iconv CMAKE_MODULE Iconv PKG_CONFIG_SPECS iconv)
           else()
-            sdl_link_dependency(iconv LIBS Iconv::Iconv CMAKE_MODULE Iconv PKG_CONFIG_LIBS iconv)
+            if (VISIONOS)
+              sdl_link_dependency(iconv LIBS iconv CMAKE_MODULE Iconv PKG_CONFIG_LIBS iconv)
```

Comment:
Doesn't cmake's FindIconv.cmake work for visions?

Reply:
It didn't appear to. I reported it CMake (https://gitlab.kitware.com/cmake/cmake/-/issues/25161) and will revert this change.